ooniprobe/Utility/TestLists.mm

```diff
@@ -0,0 +1,76 @@
+#import "TestLists.h"
+#import "NTYCSVTable.h"
+#include <measurement_kit/ooni.hpp>
+#include <measurement_kit/ooni/orchestrate.hpp>
+
+@implementation TestLists
+
++ (id)sharedTestLists
+{
+    static TestLists *sharedTestLists = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        sharedTestLists = [[self alloc] init];
+    });
+    return sharedTestLists;
+}
+
+-(id)init
+{
+    self = [super init];
+    if(self)
+    {
+        [self getCC];
+    }
+    return self;
+}
+
+- (NSArray*)getUrls{
+    NSMutableArray *urls = [[NSMutableArray alloc] init];
+    NSArray *global_urls = [self getUrlsForCountry:@"global"];
+    [urls addObjectsFromArray:global_urls];
+    NSArray *local_urls = [self getUrlsForCountry:[self.probe_cc lowercaseString]];
```

Comment:
What if `probe_cc` is `nil`?

Reply:
I also don't get exactly where is `probe_cc` initialized

Reply:
The next ```pathForResource:country_code ofType:@"csv"``` will return nil so local_urls will be an empty array